op-node/rollup/types.go

```diff
@@ -163,6 +164,9 @@ type Config struct {
 	// This feature (de)activates by L1 origin timestamp, to keep a consistent L1 block info per L2
 	// epoch.
 	PectraBlobScheduleTime *uint64 `json:"pectra_blob_schedule_time,omitempty"`
+
+	// L1 Chain Config
+	L1ChainConfig *params.ChainConfig `json:"l1_chain_config,omitempty"`
```

Comment:
From chat with @sebastianst:

We typically have 3 cases:

1. in memory test. Should have the l1 `params.ChainConfig` avaiable
2. devnets. Should have the JSON encoded l1 chain config available
3. production, likely don't need the full config, can be looked up by chain iD

Therefore it may be nicer to pass this in as a separate (optional) CLI flag, and optional constructor for op-node.